rom/laghos.cpp

```diff
@@ -1727,8 +1822,81 @@ int main(int argc, char *argv[])
         }
     }
 
+    double errorIndicator = INT_MAX;
+    bool errorIndicatorComputed = false;
+    int errorIndicatorVecSize = 0;
+
     if (rom_online)
     {
+
+        // If using the greedy algorithm, calculate the error indicator using the FOM lifted during
+        // the second to last step compared against the FOM lifted at the last step.
+        if (rom_build_database && !rom_calc_rel_error)
+        {
+            basis[romOptions.window]->LiftROMtoFOM(romS, *S);
+            if (romOptions.greedyErrorIndicatorType == useLastLiftedSolution)
+            {
+                if (converged)
+                {
+                    Vector errorIndicatorVec = Vector(lastLiftedSolution.Size());
+                    subtract(lastLiftedSolution, *S, errorIndicatorVec);
+
+                    errorIndicator = errorIndicatorVec.Norml2();
+                    errorIndicatorVecSize = errorIndicatorVec.Size();
+                }
```

Comment:
What happens in the `!converged` case, when `errorIndicator == INT_MAX`?

Reply:
The greedy algorithm takes this information and sets the error indicator to INT_MAX, meaning we definitely need to build a ROM near that point.